bigquery/deploy/README.md

```diff
@@ -42,9 +42,9 @@
     This will make `kubectl` use this cluster.
 
 * Run Elasticsearch on GKE
```

Comment:
This doesn't have to be done now, but I imagine eventually this section will also move to either the k8s-es-cluster subdirectory or the project root README and each indexer deploy documentation will link out to setting up Elasticsearch.